plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/utils/PlayAssetDeliveryUtilsTest.java

```diff
@@ -0,0 +1,63 @@
+package com.google.play.core.godot.assetpacks.utils;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.android.play.core.assetpacks.AssetPackState;
+
+import org.godotengine.godot.Dictionary;
+import org.junit.Test;
+
+public class PlayAssetDeliveryUtilsTest {
```

Comment:
I had some near duplicate code here comparing to convertAssetPackStateToDictionary(), but it seems unavoidable.

Reply:
You could make `constructAssetPackStateDictionaryHelper` a private method in utils and then leverage it in `convertAssetPackStateToDictionary`:
````
public static Dictionary convertAssetPackStateToDictionary(AssetPackState assetPackState) {
   return constructAssetPackStateDictionaryHelper(assetPackState.bytesDownloaded, assetPackState.errorCode, etc...);
}
````

Reply:
I refactor ```constructAssetPackStateDictionary``` to 
```
public static Dictionary constructAssetPackStateDictionary(
      long bytesDownloaded,
      int errorCode,
      String name,
      int status,
      long totalBytesToDownload,
      int transferProgressPercentage);
```
This helper function is public since I also need to access it in PlayAssetDeliveryUtilsTest class